### PR TITLE
Add G2A

### PR DESCRIPTION
**Pull request by Reprevise**

### CODE REVIEW

**Comment by karson111** [comment]
Can you add mineplex.com since it has email 2fa authenticater app, and backup codes.

// Edit @kmpoppe on 20.09.19 14:26 CEST: #4023 was closed as the requested Site is not within the scope of what we list on twofactorauth.org //

File: _data/gaming.yml [hunk 1]
```diff
@@ -98,6 +98,14 @@ websites:
       - software
     doc: https://support.faceit.com/hc/en-us/articles/115000254670
 
+  - name: G2A
+    url: https://www.g2a.com
+    img: g2a.png
+    tfa:
+	  - software
```
Comment (kmpoppe) [inline _data/gaming.yml]:
```suggestion
      - software
```

File: _data/gaming.yml [hunk 2]
```diff
@@ -98,6 +98,14 @@ websites:
...
+	  - sms
```
Comment (kmpoppe) [inline _data/gaming.yml]:
```suggestion
      - sms
```

**Comment by Reprevise** [comment]
G2A is a popular game re-selling website that handles money which is within the scope of TwoFactorAuth.org.

**Comment by RichJeanes** [comment]
G2A also has a very questionable reputation for being a marketplace for illegally obtained game keys that all but refuses to work with publishers and developers to curtail the problem. Don't know if that should be taken into account when determining its listability.

**Comment by kmpoppe** [comment]
This PR is closed because G2A has the potential to damage the reputation of twofactorauth.org. Consequently, according to our [Exclusion Guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md#potential-controversial-sites), the site shall not be listed.
If proof arises that this assumption is wrong, a new PR could be opened.

**Pull request closed without merging**